Automattic/vip-design-system
# PIE-3831 Fixes word wrapping on OptionRow with meta

## Description

### Before

![localhost_6006__path=_story_optionrow--with-meta](https://github.com/Automattic/vip-design-system/assets/156388/a8272882-9e6e-4a06-9bae-7a082536c797)

### After

![localhost_6006__path=_story_optionrow--with-meta (1)](https://github.com/Automattic/vip-design-system/assets/156388/c056584f-8822-46a3-9e5a-b69bfff096e0)

## JIRA
- https://vipjira.atlassian.net/browse/PIE-3831

## Checklist


## Steps to Test

1. Pull down PR.
1. `npm run dev`.
1. Open Storybook.
1. Go to the OptionRow -> With Meta storybook example
1. Resize window to the width of 420px
1. Verify the meta text doesn't overlap with the body text
